### PR TITLE
Grey span on settings

### PR DESCRIPTION
**Pull request by Kledis98**

### CODE REVIEW

File: src/app/settings/settings.component.css [hunk 1]
```diff
@@ -129,3 +129,13 @@
     background-color: #797979;
     box-shadow: inset -7px -3px 0 0 #ffffff;
 }
+.dark-mode-informative-text{
+    padding-top: 0.3rem;
+    font-size: 0.8rem;
+    color: rgb(180, 180, 180);
```
Comment (OpenCovenant) [inline src/app/settings/settings.component.css:135]: Perhaps use hex colors? Think we use that basically everywhere else. Same for line 140 here.

File: src/app/header/header.component.css [hunk 2]
```diff
@@ -183,13 +183,13 @@ hr {
     color:#858585;
 } */
 
-.dark-mode-theme {
+.dark-mode-informative-text {
     padding: 1rem;
     padding-top: 0.3rem;
     font-size: 0.8rem;
     color: rgb(180, 180, 180);
 }
```
Comment (OpenCovenant) [inline src/app/header/header.component.css:191]: Add a new line below, same for line 136 in `src/app/settings/settings.component.css`.

File: src/app/settings/settings.component.css (hunk 1 above)
Comment (AndiBraimllari) [inline src/app/settings/settings.component.css:135]: Perhaps use hex colors (here and in other changed lines)? Think we use that basically everywhere else.

File: src/app/header/header.component.css (hunk 2 above)
Comment (AndiBraimllari) [inline src/app/header/header.component.css:191]: Add a new line below, same for line 136 in `src/app/settings/settings.component.css.`

**Review by AndiBraimllari — APPROVED** [review]

**Comment by Kledis98** [comment]
Closing this PR, because grey span on settings is added on another one. 
Solved!

**Pull request closed without merging**